lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
It might make sense to group this together with `NonlinearityLayer`... not sure that really belongs in `dense.py` as it is currently.